libs/api/mocks/src/domains/cms/store.ts

```diff
@@ -23,19 +23,19 @@ export const store = createStore(() => {
 
   // Create articles, groups and subgroups in every category.
   const articles = articleCategories.reduce<Article[]>((articles, category) => {
-    const groups = articleGroup.list(faker.datatype.number({ min: 2, max: 4 }))
+    const groups = articleGroup.list(faker.number.int({ min: 2, max: 4 }))
 
     for (const group of groups) {
       const subGroups =
-        faker.datatype.number(4) === 0
+        faker.number.int(4) === 0
           ? []
           : articleSubgroup.list(faker.datatype.number({ min: 2, max: 4 }))
```

Comment:
```suggestion
          : articleSubgroup.list(faker.number.int({ min: 2, max: 4 }))
```

Reply:
I already fixed this in 04b53b2a0bdd865ee62efeb6bbdfc2bb22f850b8. You might need to refresh your browser 🤔